apps/src/studio/studio.js

```diff
@@ -5266,11 +5266,19 @@ Studio.vanishActor = function(opts) {
 };
 
 Studio.setSpriteEmotion = function(opts) {
-  Studio.sprite[opts.spriteIndex].emotion = opts.value;
+  let sprite = Studio.sprite[opts.spriteIndex];
+  if (sprite === undefined) {
```

Comment:
nit / your choice: simplify to `if (!sprite) {}`

Reply:
I'm going to leave it as is because I think we've previously had a bug where the value was falsy, but not `undefined`,  and we checked `!sprite` when what we should've checked `sprite === undefined`, so I'm leaning towards being more explicit here.